docs/controller/ingress/example.yml

```diff
@@ -58,9 +58,9 @@ properties:
     # Here we have a static route that is always present.
     Routes:
       - LoadBalancerPort: 80
-        LoadBalancerProtocol: https
+        LoadBalancerProtocol: HTTP
```

Comment:
Did this become case sensitive at some point?   Curious why you'd change it to all caps.

Reply:
Let me go back and look at this one as I made it a while ago. I recall at the time that the check between what is in the yml and what the Routes API returned for protocol was case sensitive, and this might have been fixed. I moved to the Protocol case in protocol.go.